app/_hub/critical-blue/approov/index.md

```diff
@@ -4,7 +4,6 @@ publisher: CriticalBlue Ltd
 
 categories:
   - authentication
-  - security
```

Comment:
Why is being removed from security?

Reply:
Hi Paulo, You beat me to contacting you about this. We are in a Plugin meeting right now updating plugins, and it seems we can only have plugins listed under one category on the Plugin Hub. You can list the plugin under Security or Authentication -- and we thought Authentication would be best? But please let me know what you prefer. The template allows you to list two categories, but we don't want the plugin to display twice on the hub.

Reply:
Also, we removed the "incompatible" section, as we are no longer listing incompatible versions. We are only listing compatible versions for both CE and EE. The display was confusing when incompatible versions were listed (as they looked as if they were supported), so we have since removed these to make it more obvious what versions are supported.

Reply:
We realized your plugin was the only plugin listed under two sections on the Hub, which is why we are only allowing it to be listed under one section = either Security or Authentication. Hope this is okay with you, and please let me know what section to list the plugin. 
Note..we are in the process of updating our template/process for adding plugins to our hub, which will make all this more clear when we make this information available.

Reply:
Thanks for the detailed feedback.

Given the fact that we can only be in one of the sections we prefer to be listed under `Security`.